brownie/convert/datatypes.py

```diff
@@ -9,6 +9,7 @@
 except ImportError:
     DecimalOverrideException = BaseException  # regular catch blocks shouldn't catch
 
+import cchecksum
```

Comment:
can it be optional? try importing and fall back to eth_utils version if not installed.